### DISCUSSION TITLE
How to suggest additional tags for existing pack?

### DISCUSSION THREAD

**Discussion by mziehm** [signalstickers/signalstickers#862] [Ideas / feature request]
How to suggest additional tags for existing pack?

One way could be a little button on each pack leading to a page similar to the contribution page, but with just a Tags field and the anti-robot question (and internally prefilled with the pack ID).

**Comment by romainricard** [comment]
Thanks for the suggestion! There is no dedicated feature to do this (yet?). For the moment, the best thing to do is open an Issue here, or send me an e-mail!

**Reply by mziehm** [reply]
see #864 - hope that works

**Comment by Meteor0id** [comment]
I think developing a system for tag suggestions would very much improve sticker search on the platform
